pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -598,13 +605,67 @@ func (r *rbac) ensureClusterRoleBindings(ctx context.Context, sa *corev1.Service
 		return r.updateClusterRoleBinding(ctx, viewCRB, sa)
 	}
 
+	go func() {
```

Comment:
Isn't this at risk of running multiple times ? (and thus create multiple infinite loop — that could be a leak)

Reply:
running this in goroutine because delete events are not coming 
:thinking:

Reply:
wdym delete events are not coming? if a namespace is deleted reconciler is not getting triggered?

Reply:
Yes it was my mistake 
Actually events are coming but i added my changes under namespaces range and because of this condition https://github.com/tektoncd/operator/blob/main/pkg/reconciler/openshift/tektonconfig/rbac.go#L191-L193 from second iteration the call is not going further

I have updated now and its working as expected

PTAL